typing/typedecl.ml

```diff
@@ -2367,31 +2375,43 @@ let add_types_to_env decls shapes env =
       add_type ~check:true ~shape id decl env)
     decls shapes env
 
+
+(* Note [Quality of jkinds during inference]
```

Comment:
This is great. Please refer to this Note (by name) from the calls to `mark_best`.

Reply:
done in the commit that added the note.